content/en/docs/reference/glossary/graceful-eviction.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Graceful eviction
+id: graceful-eviction
```

Comment:
I want to use the word “exogenous”, but it's too complicated.
```suggestion
title: API-initiated eviction
```
How about this?